index1.js

```diff
@@ -1,4 +1,17 @@
-var userName1=prompt("What is your name?")
-var userName2=prompt("how old are you?")
-var userName3=prompt("What city are you coming from?")
-alert(`${userName1} ` + `${userName2} ` + `${userName3} `)
\ No newline at end of file
+// //  скрипт для додавання, віднімання, множення та поділу двох чисел
+
+let userNum =+prompt('enter any number');
```

Comment:
It will be easier to read)

```suggestion
let userNum =Number(prompt('enter any number'));
```